jstz_cli/src/main.rs

```diff
@@ -5,10 +5,12 @@ mod account;
 mod bridge;
 mod config;
 mod deploy;
+mod logs_command;
```

Comment:
`logs_command` -> `logs`, fits better w/ existing naming

Reply:
The main reason why I didn't like it was that logs is a special folder in VS Code making it a bit confusing.

Reply:
And it is by default ignored in .gitignore